examples/jf-admin-app/linux/rpc/RpcServer.cpp

```diff
@@ -26,52 +26,59 @@ ::pw::Status JointFabric::TransferOwnership(const ::OwnershipContext & request,
         ChipLogError(JointFabric, "Invalid ICAC Public Key Size");
     }
 
+    if (request.jcm && request.peerAdminJFAdminClusterEndpointId == kInvalidEndpointId)
+    {
+        return pw::Status::OutOfRange();
+        ChipLogError(JointFabric, "Invalid Peer Admin Endpoint ID for the JF Administrator Cluster");
```

Comment:
is this log message reachable?

Reply:
thx for this, somehow I wrongly switched the order of the two lines.